qiita_pet/templates/user_messages.html

```diff
@@ -0,0 +1,10 @@
+{% extends sitebase.html %}
+{% autoescape None %}
+{% block content%}
+<h2 style="text-decoration: underline;">System Messages</h2>
+<ul>
```

Comment:
No it's not. The indentation in the html files are 2 spaces. This is how it should look (accordingly to other files in qiita)
```html
<ul>
  {% for msg in messages %}
    <li>{% raw msg %}</li>
  {% end %}
</ul>
```

Also note that I think this should be `raw msg` instead of only `msg` since the msg is in html format, correct?